dobi.yaml

```diff
@@ -6,3 +6,73 @@ image=image-kas:
   image: ghcr.io/siemens/kas/kas
   tags: ['2.3.3']
   pull: once
+
+image=image-curl-jq:
+  image: dwdraju/alpine-curl-jq
+  tags: [latest]
+  pull: once
+
+image=image-mender-cli:
+  image: ci4rail/mender-cli
+  tags: [master]
+  pull: once
+
+# ===================================================
+# mounts
+# ===================================================
+
+# Where the yocto-build happens
+mount=build-dir-yocto:
```

Comment:
duplicated "yocto" mounts

Reply:
Yes, but this is necessary. Kas expects the repos in /repo/src (which is ./yocto), and the rest of build system uses /work (which is also ./yocto)